shared/templates/grub2_bootloader_argument/ansible.template

```diff
@@ -4,32 +4,32 @@
 # complexity = medium
 # disruption = low
 
-{{% if product in ["rhel7", "ol7"] or 'ubuntu' in product %}}
-- name: check {{{ ARG_NAME }}} argument exists
+{{% if product in ["rhel7", "ol7", "rhel9"] or 'ubuntu' in product %}}
```

Comment:
It looks like this change needs to be applied for bash/OVAL as well.

I found out that RHEL8 STIG recommends using `grubby` to set kernel parameters, I wonder if it makes sense to include rhel8 in this list: https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_8_security_technical_implementation_guide/V-230279

Even though this started as a change a long time ago because RHEL7 and RHEL8 were configured in a different way: 
https://github.com/ComplianceAsCode/content/pull/4786

@yuumasato @Mab879 @vojtapolasek

Reply:
https://github.com/ComplianceAsCode/content/pull/7931